assets/dev/js/frontend/elements-handlers-manager.js

```diff
@@ -102,6 +104,8 @@ module.exports = function( $ ) {
 
 		const newHandler = new HandlerClass( options );
 
+		elementorFrontend.hooks.doAction( `frontend/element_handler_ready/${ options.elementName }`, options.$element, $ );
```

Comment:
BTW: Why not trigger a native event?

```suggestion
		elementorFrontend.hooks.doAction( `frontend/element-handler-ready/${ options.elementName }`, options.$element, $ );
```

Reply:
@matipojo - Are you sure? all the other actions are with underscore: `frontend/element_ready/${ widgetType }`, `panel/open_editor/${ elementType }/${ widgetType }`

Reply:
@matipojo - As agreed with @rami-elementor it should be with an underscore to match the element_ready action.